src/server/index.js

```diff
@@ -0,0 +1,31 @@
+const { createServer } = require( 'http' );
+const healthCheckUrl = '/cache-healthcheck?';
+
+module.exports = ( { PORT = 3000, requestHandler, express = false, logger = console } ) => {
```

Comment:
It would be cool if the function accepted a single argument (an express app), or the option object like we have. I would imagine 99% of use-cases will be wrapping an express app, so being able to do:

```
import go from 'vip-go-node';

const app = express();

const server = go( app );
```

without the more verbose object notation. Then if extra config is necessary, one could do:

```
const server = go( {
    requestHandler: app,
    port: 3000,
    logger,
} );
```

Reply:
Can we default the logger to the VIP logger package? Need to consider how that behaves in both production and development.

Reply:
I'm not sure if there is a way to detect an `express` app 🤔 . However, given than the most usage will be with an `express` app, I think defaulting to an express app will be the perfect solution.

For the VIP logger, we need to initialize it with a namespace. What namespace could we use in this case?

cc @nickdaugherty , @mjangda

Reply:
Maybe it's an instance of a class? Or there is probably a 'version' property or something else that's unique to the the express server.

Reply:
I double checked and couldn't find something unique. Running `Object.getPrototypeOf(expressApp)` returns `function() { native code }` for both the express app and the custom handler.

One way we can do it is trying to access `app.route`, the express app will work but the custom handler will not, given that it should be a function. So naturally, no `.route` (in general `.something`) will work. What do you think?